av/video/reformatter.pxd

```diff
@@ -9,4 +9,5 @@ cdef class VideoReformatter(object):
 
     cdef _reformat(self, VideoFrame frame, int width, int height,
```

Comment:
Is it possible to enforce keywords-only arguments in Cython? With the ever-increasing number of options I'm afraid this is going to turn into a footgun. Thoughts @mikeboers ?

Reply:
.. and my question was a bit stupid, `_reformat` is purely internal, the user-facing API is `reformat` which is just regular Python